code/datums/status_effects/buffs.dm

```diff
@@ -436,3 +436,8 @@
 	name = "Fleshmend"
 	desc = "Our wounds are rapidly healing. <i>This effect is prevented if we are on fire.</i>"
 	icon_state = "fleshmend"
+
+/datum/status_effect/exercised
```

Comment:
Status effects process via SSfastprocess, an effect that lasts 20 minutes doesn't need it, please override as necessary